### PR TITLE
Add filetype clarification for notifications (iOS / Android)

### PR DESCRIPTION
**Pull request by rwarner**
Have a few different changes in here resolving some confusion I had with notification usage last night for Android.

Didn't realize that movie/audio filetypes weren't support, so I went ahead and added some clarifications into this page to help others. Let me know if there's some changes suggested. I went ahead and tried to make it look the best and not too intrusive.

Am open to suggestions or other locations for clarifications.

Capture2

Further down....

Capture

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @rwarner, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;companion.home-assistant#431).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: docs/notifications/attachments.md [hunk 1]
```diff
@@ -75,19 +76,25 @@ The same notification but expanded to show the full size image attachment:
...
+| :-------: | --------------- | ------------------|------------------------- |
+|    Image    | 10 MB    | JPEG, GIF, PNG          | ![Android](/assets/android.svg) Android & ![iOS](/assets/iOS.svg) |
+|   Video   | 50 MB   | MPEG, MPEG2, MPEG4, AVI   | ![iOS](/assets/iOS.svg) |
+|   Audio    | 5 MB  | AIFF, WAV, MP3, MPEG4 Audio          | ![iOS](/assets/iOS.svg) |
 
 
 ## Configuration
 ![iOS](/assets/iOS.svg)Specific<br />
+
+ [`See table for supported media`](#supported-media-types)
```
Comment (TomBrien) [inline docs/notifications/attachments.md:89]:
```suggestion
 [See table for supported media](#supported-media-types)
```

File: docs/notifications/attachments.md [hunk 2]
```diff
@@ -75,19 +76,25 @@ The same notification but expanded to show the full size image attachment:
...
+
 -   **url** (*Required*): The URL of content to use as the attachment. This URL *must* be accessible from the Internet, or the receiving device must be on the same network as the hosted content.
 -   **content-type** (*Optional*): By default, the extension of the URL will be checked to determine the filetype. If there is no extension/it can't be determined you can manually provide a file extension.
 -   **hide-thumbnail** (*Optional*): If set to `true` the thumbnail will not show on the notification. The content will only be viewable by expanding.
 
 ![Android](/assets/android.svg) Android Specific
 
+ [`See table for supported media`](#supported-media-types)
```
Comment (TomBrien) [inline docs/notifications/attachments.md:97]:
```suggestion
 [See table for supported media](#supported-media-types)
```

**Review by TomBrien — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Only very slight changes from me, we should save the back tick formatting for in-line `code` and similar. 

Will let someone from the Android side approve and merge as I am unable to confirm these.

**Comment by rwarner** [comment]
Sounds good, just committed your suggestions. I figured as much that would be reserved for those things. Appreciate the help

**Review by dshokouhi — APPROVED** [review]
Looks good to me

**Pull request merged**